src/pages/Products/PC/FilterPC.tsx

```diff
@@ -0,0 +1,253 @@
+import { FilterButton } from "../../../layout/components/buttons/FilterButton";
+
+export const FilterPC = () => {
+  return (
+    <div className="card col-2 mt-2">
+      <h5 className="card-header">Filters:</h5>
+      <ul className="list-group list-group-flush">
+        <li className="list-group-item">
+          <h6 className="card-title">Processor brands</h6>
+          <div className="form-check">
+            <input
+              className="form-check-input"
+              type="checkbox"
+              value=""
+              id="intelCheck"
+            />
+            <label className="form-check-label" htmlFor="flexCheckDefault">
+              Intel
+            </label>
+          </div>
+          <div className="form-check">
+            <input
+              className="form-check-input"
+              type="checkbox"
+              value=""
+              id="amdCheck"
+            />
+            <label className="form-check-label" htmlFor="flexCheckChecked">
+              AMD
+            </label>
+          </div>
+        </li>
+        <li className="list-group-item">
+          <h6 className="card-title">Graphics card brands</h6>
+          <div className="form-check">
+            <input
+              className="form-check-input"
+              type="checkbox"
+              value=""
+              id="intelCheck"
```

Comment:
Czy tu nie należałoby zrobić porządku z `id'kami`? Bo widzę, że tutaj np. przy Nvidii, albo poniżej w innych miejscach np. RAM capacity nadal te `id` sie powtarzają

Reply:
Zrobiłem unikatowe ID, tam gdzie był processor dodałem Cpu, gdzie grafika Gpu, Ram to Ram, dysk twardy Drive